xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2240,7 +2240,7 @@ admin.colortheme.manage=Manage color themes\u00BB
 admin.customize=Customize
 admin.save=Save
 admin.defaultwikinotinstalled=Your wiki seems empty. You may want to import the default XWiki Enterprise wiki which contains a set of useful pages: user profiles, recent activity, administration pages and many more. This wiki is distributed as a XAR file, you can download it from {0}.
-admin.adminappnotinstalled=The administration application is not installed. Since XWiki Enterprise 1.5 the Administration is distributed as an application. You can download it from {0}.
+admin.adminappnotinstalled=The administration application is not installed. Since XWiki Enterprise 1.5, the Administration is distributed as an application. You can download it from {0}.
```

Comment:
Outside of scope for GCI:

Since we're now at XWiki 9+ I think we should change this message to: "The administration application is not installed. You should install it with the {0}Extension Manager{1}" and make "Extension Manager" point to the EM page. This means deprecating the previous key and creating a new one probably since the parameters would be different. WDYT?